android/key.enc.properties

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
I thought key.properties is just a set of key value properties, but the content is a json. is the output of decryption a properties file?

Reply:
This is the intended behaviour of sops, when encrypting a binary file it will store the file in the 'data' field, the other JSON values are the metadata for the encrypted file, eg. keys used to encrypt the file.

When the file is decrypted it will once again be a binary file, in this case a set of key value properties.

Reply:
Cool thanks for clarifying